commerce/domain/orchestrators/classes/CartCalculateExecutorForUnitTest.cls

```diff
@@ -10,7 +10,12 @@ global class CartCalculateExecutorForUnitTest extends CartExtension.CartCalculat
 
     global override void prices(CartExtension.CartCalculateCalculatorRequest request) {
         CartExtension.Cart cart = request.getCart();
-        cart.setName('CartRepriced');
+
+        if (cart.getStatus() == CartExtension.CartStatusEnum.CHECKOUT) {
```

Comment:
@jbisaSF Use the buyer action rather than cart.getStatus.  Cleaner

Reply:
We don't have access to `BuyerActions` through `CartExtension.CartCalculateCalculatorRequest` today. To ensure that the right behavior is invoked (consistently) when the Cart is in Checkout, we either have to:
    - Rely on Cart Status
    - Or rely on `BuyerActions`

But, I think exposing both `BuyerActions` _and_ `BuyerActionDetails` to the calculators is more confusing.